src/main/java/org/opentripplanner/street/model/edge/StreetEdge.java

```diff
@@ -1183,6 +1183,10 @@ private StateEditor doTraverse(State s0, TraverseMode traverseMode, boolean walk
       s1.incrementWalkDistance(getDistanceWithElevation());
     }
 
+    if (traverseMode.isCyclingIsh() && s1.isVehicleRentable()) {
```

Comment:
Why only for cyling-ish? Cars have a range as well.

Reply:
Removed the check for cycling-ish